app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -28,17 +28,17 @@
 import com.igalia.wolvic.speech.SpeechServices;
 import com.igalia.wolvic.telemetry.TelemetryService;
 import com.igalia.wolvic.utils.StringUtils;
+import com.igalia.wolvic.utils.SystemUtils;
 
 public class PlatformActivity extends Activity implements SurfaceHolder.Callback {
-    public static final String TAG = "PlatformActivity";
-
+    private static final String LOGCAT = SystemUtils.createLogtag(PlatformActivity.class);
```

Comment:
I think there is a misspelling here, we normally call this `LOGTAG`

In any case I don't think we need to do any change here now as it's unrelated to the kiosk mode